subgraphs/aave-forks/protocols/aave-v3/README.md

```diff
@@ -24,7 +26,7 @@ Portion of the Total Revenue allocated to the Protocol
 
 Sum across all Pools:
 
-`(Pool Oustanding Borrow Amount * Pool Borrow Rate) * (Pool Reserve Factor)`
+`(Pool Oustanding Borrow Amount * Pool Borrow Rate) * (Pool Reserve Factor) + liquidiated collateral amount * liquidiation penalty * liquidation procotol fee percentage`
```

Comment:
getting there.

Reply:
good point! I agree it is better to not double counting flashloan revenue to LP. For revenue details, we are [not providing   detailed FeeType for supply side revenue currently anyway](https://github.com/messari/subgraphs/blob/3c01f4df091a0fba1b435eccfa8a19626586eb14/subgraphs/_reference_/src/sdk/protocols/lending/constants.ts#L83), right.

Reply:
To me `PROTOCOL_FEE` is from interest generated, we already split it up between supply/protocol side. The fee type should be looked at as the source type of the interest/revenue/fees

Reply:
so, it'll not be a problem for flash loan revenue details to the supply side because we are not breaking supply side revenue by interest/floash loan etc, and the fee type will all be PROTOCOL_FEE for supply side revenue?
```
export namespace FeeType {
  export const LIQUIDATION_FEE = "LIQUIDATION_FEE";
  export const ADMIN_FEE = "ADMIN_FEE";
  export const PROTOCOL_FEE = "PROTOCOL_FEE";
  export const MINT_FEE = "MINT_FEE";
  export const WITHDRAW_FEE = "WITHDRAW_FEE";
  export const OTHER = "OTHER";
}
```

Reply:
So to answer the question, yes. I will check w our ppl internally and ensure this is okay

Reply:
Actually, how would you go about ensuring the rev to LP from flashloans gets added to `supplySideRev`, but we still add it to the RevenueDetails as type `FLASHLOAN` (you can add this enum type), and avoid double counting?

I think that is what we should do tbh. Sorry for all the confusion

Reply:
Okay, sorry for the confusion.

We want to be as specific as possible on aave, so let's add a new `FeeType` called `FLASHLOAN_LP_FEE` and `FLASHLOAN_PROTOCOL_FEE`, then whatever you need to do in order to separate the 2 without double counting. And just to be transparent we should mark the methodology either in a comment in the code or readme

Reply:
Yes, we can do that. I will add a helper entity/field to avoid double counting.

Reply:
This is now addressed in https://github.com/messari/subgraphs/pull/1993/commits/4421fdcc81003bfedcf6730fb57da6249c4b98f7